pubkey_encrypt.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Pubkey Encrypt
+description: Adds Password-based Public Key Encryption support.
+core: 8.x
+type: module
+package: System
```

Comment:
The "package" or module group should definitely not be "System".  I'd prefer to see "Encryption" as used by the other encryption modules (field_encrypt and encrypted_files).  Encrypt and Key are both using "Security", which I think is too generic.  It would be nice if we could place those in the "Encryption" group/package as well.